packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.types.ts

```diff
@@ -126,6 +126,13 @@ export interface IHoverCardProps extends React.HTMLAttributes<HTMLDivElement> {
    */
   target?: HTMLElement | string | null;
 
+  /**
+   * Optional target element to set event listener.
+   * This props is to separate the target to anchor hover card from the target to attach event listener.
+   * When no `eventListenerTarget` given, HoverCard will use `target`props or its root to set event listener.
+   */
```

Comment:
`@defaultvalue false`

Reply:
The type of the new prop is not a boolean. I consider no default is needed because it’s an optional prop and an explanation is given in case the user it’s not providing one.

Reply:
True, I don't know why I even wrote this. I'll resolve this comment. Thanks @Vitalius1!